### DISCUSSION TITLE
I have noticed that stable diffusion can load checkpoints about 5 time faster than forge!?

### DISCUSSION THREAD

**Discussion by MohammadAboulEla** [lllyasviel/stable-diffusion-webui-forge#667] [General]
while "stable-diffusion-webui-forge" is way faster in generating images comparing to "stable-diffusion-webui". I have noticed that stable diffusion especially in the last version 1.9.0 can load checkpoints about 5 time faster than forge. like on my device changing checkpoint or loading the first one require a bout 4.5 to 5 min while in  stable diffusion I can change checkpoints in about 50 sec.

and yes I prefer forge and the generation speed over loading but I was wondering why forge is not that good in loading checkpoint like stable diffusion!.

device info
CPU: i9 12900k
GPU: RTX 3070ti
RAM: 32 GB

**Comment by willhsmit** [comment]
Might be useful to describe your device with this report -- my RTX 3060 swaps SDXL checkpoints in 5-15s on forge and webui is about 50% slower so whatever you're describing might be device-specific.

**Reply by MohammadAboulEla** [reply]
device info
CPU: i9 12900k
GPU: RTX 3070ti
RAM: 32 GB

**Comment by MaestroFury** [comment]
are you using a disk harddrive? I'm using M.2 drives directly connected to my MB and checkpoint loading times for me in forge is 4-5 seconds, and 10-12 in auto. These are SDXL.

**Comment by freecoderwaifu** [comment]
Test with this option enabled and disabled on both A1111 and Forge.

![298793325-11ac6c52-afd4-49ee-9989-6d456b536bdb](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/66507343/161fbd5e-1df5-4f64-860a-5a03a9faa32f)

**Reply by psydok** [reply]
It doesn't work :(

**Comment by ikoseu** [comment]
I updated forge from an older version and before I could test a prompt with several different checkpoints in around 25 min, now it says it takes over an hour.. I suspect they  either broke something or some option got changed in the update.

I am testing right now disabling and enabling memmapping as suggested by the user above.. lest see..
